src/Simplex/Chat/Controller.hs

```diff
@@ -198,6 +199,9 @@ data ChatCommand
   | ListUsers
   | APISetActiveUser UserId (Maybe UserPwd)
   | SetActiveUser UserName (Maybe UserPwd)
+  | SetAllContactReceipts Bool
+  | APISetUserContactReceipts UserId UserMsgReceiptSettings
+  | SetUserContactReceipts UserMsgReceiptSettings
```

Comment:
will there be per contact, per group api?

Reply:
we have API to update contact settings - it will be used